src/landscape/components/LandscapeForm/BoundaryStep.js

```diff
@@ -113,6 +117,9 @@ const MapDrawPolygon = props => {
       ...landscape,
       areaPolygon,
     });
+    trackEvent('Landscape created', {
```

Comment:
Issue: I don't think this is necessarily creating an event after the landscape is created, it is called when the request is sent, after this still there could be backend validations, I am not sure how the function that calls the API results is propagated within that `save()` function but ideally you shoul be able to check it save was succesful using promises `save(...).then(() => track even)`

Reply:
Oh yes good point.